CHANGES.rst

```diff
@@ -49,6 +51,9 @@ API Changes
 Cubeviz
 ^^^^^^^
 
+- ``spatial_subset`` in the spectral extraction plugin is now renamed to ``aperture`` and will
+  be removed in a future release. [#2664]
```

Comment:
Want to be clearer that the feature will not be removed, just the old name, yes?
```suggestion
- ``spatial_subset`` in the spectral extraction plugin is now renamed to ``aperture`` and the deprecated name will
  be removed in a future release. [#2664]
```